### DISCUSSION TITLE
GitHub Actions Inquiry

### DISCUSSION THREAD

**Discussion by go-christianharrel** [PUP-BSIT/playground-group-vrzn_appdev#22] [Q&A] [answered]
Hi, can someone please help me with on how to deploy Angular App to Firebase Hosting. Thank you!

**Comment by Galope-Jhean** [comment, marked as answer]
Check this pipeline out

https://github.com/PUP-BSIT/playground-group-vrzn_appdev/blob/main/.github/workflows/main.yml

**Reply by go-christianharrel** [reply]
I'll try to follow this guide. Thanks for this!

**Comment by go-christianharrel** [comment]
aaa
